tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacade/BraintreeFacadeAuthorizeTest.php

```diff
@@ -30,34 +27,42 @@ class BraintreeFacadeAuthorizeTest extends AbstractFacadeTest
     /**
      * @return void
      */
-    public function testAuthorizePaymentWithSuccessResponse()
+    public function testAuthorizePaymentWithSuccessResponse(): void
     {
+        // Arrange
         $factoryMock = $this->getFactoryMock(['createAuthorizeTransaction']);
         $factoryMock->expects($this->once())->method('createAuthorizeTransaction')->willReturn(
             $this->getAuthorizeTransactionMock(),
         );
         $braintreeFacade = $this->getBraintreeFacade($factoryMock);
-
         $transactionMetaTransfer = $this->getTransactionMetaTransfer();
+
+        // Act
         $response = $braintreeFacade->authorizePayment($transactionMetaTransfer);
+        $isSuccess = $response->getIsSuccess();
```

Comment:
This can be removed, it is not used. PhpStorm should show you that in the editor.

Reply:
No change yet, did you push your changes?

Reply:
Yes, check, please.